models/DataObject/ClassDefinition/Data/Block.php

```diff
@@ -635,26 +636,6 @@ public function getLayout()
         return $this->layout;
     }
 
-    /**
-     * @param mixed $data
-     * @param array $blockedKeys
-     *
-     * @return $this
-     */
-    public function setValues($data = [], $blockedKeys = [])
```

Comment:
$blockedKeys not necessary anymore ?

Reply:
I add it direct to the parent class `\Pimcore\DataObject\ClassDefinition\Data` because of:
https://github.com/pimcore/pimcore/blob/643f7384db4a433d9f439930ecc0b6cc7f2c10a9/models/DataObject/ClassDefinition/Service.php#L266

Reply:
Saw that, but the question is if the blockedKeys parameter isn't needed. It is not supported by the parrent, right ?

Reply:
The parent class has now the blockedKeys parameter. So it is no longer necessary in this child class, as it is inherited:
https://github.com/pimcore/pimcore/blob/643f7384db4a433d9f439930ecc0b6cc7f2c10a9/models/DataObject/ClassDefinition/Data.php#L332-L350